app/helpers/agent_helper.rb

```diff
@@ -185,11 +185,22 @@ def generate_agent_tooltip(agent_icon, name, email = nil, identifiers = nil, aff
       content_tag(:div) do
         content_tag(:div, name, class: 'agent-name') +
         content_tag(:div, email || '', class: 'agent-dependency') +
-        content_tag(:div, identifiers || '', class: 'agent-dependency') +
-        content_tag(:div, affiliations || '', class: 'agent-dependency')
+        unless identifiers.to_s.empty?
+          content_tag(:div, class: 'agent-dependency') do
+            inline_svg_tag('icons/orcid.svg', class: 'agent-dependency-icon') +
```

Comment:
for agents of type "Organization" they don't use ORCID but a ROR, so in this case the Icon need to be this one 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/9a3c1aa8-5073-4e85-86fc-3840fd4e91a9)
or something like it.
this there website https://ror.org/